src/applications/mhv/secure-messaging/components/MessageList/FolderHeader.jsx

```diff
@@ -10,7 +10,7 @@ import ComposeMessageButton from '../MessageActionButtons/ComposeMessageButton';
 const FolderHeader = props => {
   const { folder, searchProps } = props;
   const location = useLocation();
-
+  let showComposeMessage = true;

```

Comment:
I would encourage using `useState()` react hook to maintain the state of this variable, where React would more reliably handle rendering of components that are conditioned by it. Like so:
`const [showComposeMessage, setShowComposeMessage] = useState(false)`
Below in your code, use setter function `setShowComposeMessage` instead of assigning a variable.

Reply:
Ok, will do that.